conveyor/adapter/id.go

```diff
@@ -0,0 +1,19 @@
+/*
+ *    Copyright 2019 Insolar Technologies
+ *
+ *    Licensed under the Apache License, Version 2.0 (the "License");
+ *    you may not use this file except in compliance with the License.
+ *    You may obtain a copy of the License at
+ *
+ *        http://www.apache.org/licenses/LICENSE-2.0
+ *
+ *    Unless required by applicable law or agreed to in writing, software
+ *    distributed under the License is distributed on an "AS IS" BASIS,
+ *    WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+ *    See the License for the specific language governing permissions and
+ *    limitations under the License.
+ */
+
+package adapter
+
+const ResponseSenderAdapterID = 1
```

Comment:
Should we assign ID in adapter in catalog on adapter registration

Reply:
Now it's not clear how to do it, since we need this ID and in adapter and in helper, but helper doesn't have access to adapter.